packages/ng/src/app/select/select.component.ts

```diff
@@ -188,6 +190,11 @@ implements ControlValueAccessor, AfterContentInit, OnInit, OnDestroy {
 						}
 				});
 			}
+
+			if (this.optionFeeder){
+				this._picker.optionFeeder = this.optionFeeder;
+				this.optionFeeder.registerKeyevent(this.onKeydown.bind(this));
```

Comment:
?

Reply:
As i said in a previous comment, when an `IOptionFeeder` is present, this one could take the focus and the input. For example, the input in a search component. In that way, the basic keyboard listener isn't working, so we have to ask to the component with the input to give us the control.